blog/2024-08-14-conda-ecosystem-explained.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: "Conda Ecosystem Explained"
+slug: "2024-08-14-conda-ecosystem-explained"
+authors: [dashagurova, jezdez]
+tags: [conda, community, governance]
+description: |
+  Get to know the conda ecosystem—a vibrant community supporting an open-source, language-agnostic, and multi-platform package management system. Discover conda CLI tools for managing packages and environments, and explore key repositories like conda-forge, bioconda and defaults. You will learn how to get started with conda tools, the important things to keep in mind, and how multistakeholder governance ensures the ecosystem remains reliable and accessible to all.
+---
+
+Get to know the conda ecosystem—a vibrant community supporting an open-source, language-agnostic, and multi-platform package management system. Discover conda CLI tools for managing packages and environments, and explore key repositories like conda-forge, bioconda and defaults. You will learn how to get started with conda tools, the important things to keep in mind, and how multistakeholder governance ensures the ecosystem remains reliable and accessible to all.
+
+<!-- truncate -->
+## What is the conda ecosystem?
+
+At its core, the conda ecosystem is a community supporting an open-source, language-agnostic, and multi-platform package management system.
+
+But what does a "package management ecosystem" really mean? Without diving too deep into every detail, here are the three main components the conda community maintains:
+
+- **CLI Tools for Package Management**: conda command-line tools make it easy to install and manage dependencies in isolated development environments across Linux, Windows, and macOS. Conda works with packages written in any language, e.g. Python, R, C, or Fortran.
+- **Package Repository**: most publicly available conda packages are hosted on **[anaconda.org](https://anaconda.org/)**, organized into channels. Channels are like remote warehouses where packages are stored and can be pulled from. You can find channels maintained by the community, companies, or individuals.
+- **Build Tools**: If you want to create and share your own conda packages, the ecosystem provides tools to help you build and distribute them efficiently.
+
+This system is bolstered by various other supporting projects, all of which you can explore on [conda's GitHub organization](https://github.com/conda) or emerging projects in the [conda-incubator](https://github.com/conda-incubator).
+
+## Conda package & environment manager
+
+The cornerstone of this entire ecosystem is the [`conda`](https://github.com/conda/conda) package and environment manager. Originally developed by Anaconda, Inc. and implemented in Python, `conda` is not limited to Python \- it's a fully capable package manager that handles dependencies across multiple languages and platforms. Whether you're a data scientist working with Python or R or you're a machine learning engineer dealing with libraries in C or Fortran, `conda` has you covered.
```

Comment:
```suggestion
The cornerstone of this entire ecosystem is the [conda](https://github.com/conda/conda) package and environment manager. Originally developed by Anaconda, Inc. and implemented in Python, conda is not limited to Python \- it's a fully capable package manager that handles dependencies across multiple languages and platforms. Whether you're a data scientist working with Python or R or you're a machine learning engineer dealing with libraries in C or Fortran, conda has you covered.
```

The usage of "conda" vs "`conda`" is not consistent; I think the style preferene is to monospace "`conda`" only when referring to a command.

Reply:
Thank you @beeankha, that's a very good callout!